src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -220,6 +226,8 @@ public virtual Task DispatchEventAsync(int eventHandlerId, EventFieldInfo fieldI
                 throw new ArgumentException($"There is no event handler with ID {eventHandlerId}");
             }
 
+            Log.HandlingEvent(_logger, eventHandlerId, eventArgs);
+
             if (fieldInfo != null)
             {
                 var latestEquivalentEventHandlerId = FindLatestEventHandlerIdInChain(eventHandlerId);
```

Comment:
Is there any value (for us to debug) to also log this?

Reply:
I don't see any definite reason to do that, but we could always add an extra log message in the future if a reason arises.